rocketpy/rocket/aero_surface/air_brakes.py

```diff
@@ -0,0 +1,207 @@
+import warnings
+
+import numpy as np
+
+from rocketpy.mathutils.function import Function
+from rocketpy.plots.aero_surface_plots import _AirBrakesPlots
+from rocketpy.prints.aero_surface_prints import _AirBrakesPrints
+
+from .aero_surface import AeroSurface
```

Comment:
Any particular reason for not using absolute path here?

```suggestion
from rocketpy.rocket.aero_surface.aero_surface import AeroSurface
```

All the other 3 imports are absolute.

Reply:
Its just easier to understand what you are importing since its from a file in the same folder